config/locales/en.yml

```diff
@@ -47,6 +47,10 @@ en:
     cy: Cymraeg
     en: English
 
+  recaptcha:
+    errors:
+      verification_failed: verification failed, please try again.
```

Comment:
`verification failed, please try again.` --> 'Verification failed, please try again.'

The first letter 'V' should be capitalised. This is in lieu of the Welsh translation also being capitalised.

Reply:
![money_advice_service](https://user-images.githubusercontent.com/7126705/32778602-7826e56e-c932-11e7-8c1d-ea0f223ec551.png)
..maybe I need to go back to translation for this.. I was utilising the fact that in our validation messages we post the validation type first (Postcode, Email) and then the error, so you see the error like above ^ 

This is the error you see in Welsh. Capitalised to try and define the start of the sentence... 
![y_gwasanaeth_cynghori_ariannol](https://user-images.githubusercontent.com/7126705/32778670-ad8a8256-c932-11e7-9fcc-81537a604f0b.png)

Reply:
It's all kind of an awkward solution as reCAPTCHA itself does not support welsh language

Reply:
@willhall88 what does UX have to say about this though? I do agree, it looks very awkward.

Reply:
who is this for a solution...? 
![y_gwasanaeth_cynghori_ariannol](https://user-images.githubusercontent.com/7126705/32780430-0c3a7602-c939-11e7-9a38-880e06e1eda9.png)

Reply:
@willhall88 yeah that looks great. Should we capitalise the English locale then?

Reply:
I'd suggest no as this will still read as a sentence as seen above

Reply:
@willhall88 ah right, that's true. The English version reads in a straigh-forward manner. Great work! Will